drivers/display/display_ili9163.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2022 Tavish Naruka <tavishnaruka@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "display_ili9xxx.h"
+#include "display_ili9163.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(display_ili9163, CONFIG_DISPLAY_LOG_LEVEL);
+
+int ili9163_regs_init(const struct device *dev)
+{
+	const struct ili9xxx_config *config = dev->config;
+	const struct ili9163_regs *regs = config->regs;
+
+	int r;
+
+
+	LOG_HEXDUMP_DBG(regs->gamset, ILI9163_GAMSET_LEN, "GAMSET");
+	r = ili9xxx_transmit(dev, ILI9163_GAMSET, regs->gamset,
+			     ILI9163_GAMSET_LEN);
+	if (r < 0) {
+		return r;
+	}
+
+	LOG_HEXDUMP_DBG(regs->gamadj, ILI9163_GAMADJ_LEN, "GAMADJ");
```

Comment:
can this copy-pasted code be rolled into a loop?

Reply:
Hi, sorry for the late response. I would prefer to do it separately later. It is similar to the other drivers, and since it is only a few commands, maybe all the similar display drivers can be improved in one go.